src/views/List.jsx

```diff
@@ -1,6 +1,13 @@
 import { ListItem } from '../components';
 
 export function List({ data }) {
```

Comment:
Maybe add a short comment about what this component does. As you build out more components and add features, adding simple documentation will help you and others (especially in the future!) have a clear idea of why this component exists.

Example:
`/** List component that displays items in a user's shopping cart  */`